### PR TITLE
stdlib('c') migration

### PR DESCRIPTION
**Pull request by n-elie**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by n-elie** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -40,6 +40,7 @@ requirements:
     - {{ stdlib('c') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
```
Comment (hmaarrfk) [inline recipe/meta.yaml:43]:
```suggestion
```

Comment (hmaarrfk) [reply recipe/meta.yaml:43]: I caught this one already

Comment (hmaarrfk) [reply recipe/meta.yaml:43]: Also please bump the build nunber

File: recipe/meta.yaml [hunk 2]
```diff
@@ -87,7 +88,6 @@ test:
     - qt6-datavis3d        # [py==310]
     - qt6-graphs           # [py==310]
 
-    - sysroot_linux-64 2.17              # [linux]
```
Comment (hmaarrfk) [inline recipe/meta.yaml:90]: While I did forget to remove this like. I'm wondering if it is worth a rebuild and push to end users. 

This migration should be "optional" and typically t be done with other work that is already occuring. 


What was the symptom that brought you here? Was something incompatible?

**Comment by n-elie** [comment]
Let's forget about this, this was done too quickly

**Pull request closed without merging**